addons/mrp/migrations/10.0.2.0/post-migration.py

```diff
@@ -397,20 +397,24 @@ def fill_stock_move_bom_line_id(cr):
         cr,
         """
         UPDATE stock_move sm_update
-        SET bom_line_id = mbl.bom_line_id
-        FROM stock_move sm
-            INNER JOIN mrp_production mp
+        SET bom_line_id = q.bom_line_id
+        FROM (
+            SELECT sm.id, mbl.bom_line_id
+            FROM stock_move sm
+            INNER JOIN mrp_production mp 
```

Comment:
```suggestion
            INNER JOIN mrp_production mp
```
delete trailing whitespace